examples/bri-3/src/bri/identity/bpiSubjects/capabilities/getBpiSubjectById/getBpiSubjectById.query.ts

```diff
@@ -0,0 +1,5 @@
+export class GetBpiSubjectByIdQuery {
+  constructor(
+    public readonly id: string,
```

Comment:
Question : why are we introducing public readonly variables? We agreed in the previous PR that variables will be private.